### DISCUSSION TITLE
Does sanbox process, inherit parent process enviroment?

### DISCUSSION THREAD

**Discussion by MrSharpp** [taskforcesh/bullmq#2043] [Q&A] [answered]
i am using a ML module with bullMQ, problem is that it loads it's models from disk.
now i imported all the models at the starting of the application, and i am expecting that it should work.

But it didn't. 

now do i manually have to write child process, logic for this kind of scenario?

**Comment by manast** [comment]
Can you write some example code and explanations on what you want to achieve and what is not working?

**Comment by MrSharpp** [comment]
the relevance of the matter to bullMQ may be limited, but I believe it would still be beneficial to hear your recommendations on how to address this situation.


- worker.ts
i am using face-api.js, and loading it's models in it.
```
faceapi.nets.ssdMobilenetv1.loadFromDisk('./weights');
faceapi.nets.tinyFaceDetector.loadFromDisk('./weights');

 const worker = new Worker(config.BULL_QUEUE_NAME, processorFilePath, {
    connection
  });
```
- job.js
now in job js i am suppose to use a function which requires that face.api model loaded.
but i cannot pass the faceapi object to the child process.

this is a blocker for me to use sandbox processors in bullMQ.

I would like you hear your suggestion for this scenario.

**Reply by manast** [reply, marked as answer]
Either load the data from inside job.js, or send the minimal data required per job or no use sandboxed processors.

**Reply by MrSharpp** [reply]
so there is no other workaround?
